.github/workflows/test_size.yml

```diff
@@ -111,15 +111,14 @@ jobs:
         shell: bash
         run: |
           python3 -m pip install -U weasyprint elf_diff anytree
-          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=plane $GITHUB_WORKSPACE/master_bin/arduplane $GITHUB_WORKSPACE/pr_bin/arduplane
-          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=copter $GITHUB_WORKSPACE/master_bin/arducopter $GITHUB_WORKSPACE/pr_bin/arducopter
           mkdir elf_diff
-          tar cf elf_diff/plane.tar plane
-          tar cf elf_diff/copter.tar copter
+          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=elf_diff/plane $GITHUB_WORKSPACE/master_bin/arduplane $GITHUB_WORKSPACE/pr_bin/arduplane
+          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=elf_diff/copter $GITHUB_WORKSPACE/master_bin/arducopter $GITHUB_WORKSPACE/pr_bin/arducopter
+	  zip elf_diff.zip elf_diff
```

Comment:
this alignement break the yml file ! that is why it doesn't run

Reply:
Thanks!